zsh/configs/post/path.zsh

```diff
@@ -1,9 +1,11 @@
 # ensure dotfiles bin directory is loaded first
 PATH="$HOME/.bin:/usr/local/sbin:$PATH"
 
-# load ASDF, falling back to rbenv if not available
-if [ -d "$HOME/.asdf" ]; then
+# load ASDF, either from home dir or brew install
+if [ -f "$HOME/.asdf/asdf.sh" ]; then
   . $HOME/.asdf/asdf.sh
+elif ASDF_PREFIX=$(brew --prefix asdf); then
```

Comment:
```
$ if y=$(brew --prefix asdf); then
> echo hi
> fi
sh: 3: brew: not found
```

I got it to be quieter with a `2>/dev/null` inside the parens.

Is it possible that `brew --prefix asdf` can produce multiple lines of output? I don't actually know what a brew is.

Also also, what do we do about the embedded newline?

All of this line relies on the return value of `brew --prefix asdf`. Is it possible to use `test`(1) here instead and set the variable outside of this line?

Reply:
Thanks Mike. I did not consider the possibility that `brew` might not be installed! (It's a package manager for the Mac, similar to `apt`). I need to rework this anyway because `brew --prefix asdf` is not a reliable way to test what I'm trying to test for. I'll look up documentation for `test`.

Reply:
Another spelling of `test` is `[` -- that's how we normally spell it in these files, but the manpage will be under `test`.